src/utils/env.ts

```diff
@@ -1 +1,4 @@
 export const isProd = () => process.env.NODE_ENV === "production";
+
+// @ts-expect-error
+export const isCypress = () => window && window.Cypress;
```

Comment:
Optional chaining is useful here:

```suggestion
export const isCypress = () => window?.Cypress;
```